src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/providers/NotcreatedDBInfluxDB.java

```diff
@@ -0,0 +1,230 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb.providers;
+
+
+import okhttp3.OkHttpClient;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.Query;
+import org.influxdb.dto.QueryResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.NumberUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class NotcreatedDBInfluxDB {
```

Comment:
This class name is confusing. At least some javadoc could help explain why it is "not creating a DB", such as:

```
/**
  This class ...
  */
```